### DISCUSSION TITLE
OpenShell windows 11

### DISCUSSION THREAD

**Discussion by ofigustavo** [Open-Shell/Open-Shell-Menu#1171] [General]
I am using windows 11 last version. Openshell does not work now. I am in preview channel, but now not working.

**Comment by auron98x23** [comment]
![image](https://user-images.githubusercontent.com/50659009/195529126-a8ac2552-8540-47f0-ba06-a74a735807b3.png)
You have this problem too? I don't know how to fix it...

**Comment by bonzibudd** [comment]
See #745
